db/migrations/0004_alter_guild_description_alter_race_name_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-10-23 14:15
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0003_alter_race_name_alter_skill_name'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='guild',
+            name='description',
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='race',
+            name='name',
+            field=models.CharField(choices=[('Elf', 'Elf'), ('Dwarf', 'Dwarf'), ('Human', 'Human'), ('Ork', 'Ork')], max_length=255, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='skill',
+            name='race',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race'),
```

Comment:
The foreign key reference 'to='db.race'' should be 'to='app_name.Race'' where 'app_name' is the name of the Django app containing the models. Ensure the correct app label is used.